### PR TITLE
Add Meetup Events using Bootstrap and jQuery

### PR DESCRIPTION
**Pull request by uncompiled**
Create Meetup JS Object that allows the user to specify a signed Meetup URL
and a maximum number of events to insert into the DOM.  Inserts into a DIV
with id="meetup-events".

### CODE REVIEW

File: meetups.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+layout: meetups
```
Comment (benjaminoakes) [inline meetups.md:2]: Do we need a layout for this?  It seems like we could just put the HTML here...

File: _layouts/default.html [hunk 2]
```diff
@@ -66,6 +67,7 @@
         <script src="js/vendor/bootstrap.min.js"></script>
 
         <script src="js/main.js"></script>
+        <script src="js/meetup-events.js"></script>
```
Comment (benjaminoakes) [inline _layouts/default.html:70]: This shouldn't hold up the merge, but we will probably want to combine these into a single `GET` request at some point.

**Comment by benjaminoakes** [comment]
I think this is good to merge, though there's some refactoring that would be nice to have in the future.  Thanks for putting your time into this!

**Comment by caseywatts** [comment]
whee! Thanks @uncompiled and @benjaminoakes  ^_^

**Pull request merged**